tests/e2e/test-build-sass.R

```diff
@@ -0,0 +1,24 @@
+rhino::build_sass()
+
+css_path <- fs::path("app", "static", "css", "app.min.css")
+
+testthat::expect_equal(
+  readLines(css_path), ""
+)
+
+# Create minimal css and check app.min.css output
+cat(
+  ".myClass { color: #fff; }\n",
+  file = fs::path("app", "styles", "main.scss"),
+  append = TRUE
+)
+rhino::build_sass()
+testthat::expect_equal(
+  readLines(css_path),
+  ".myClass{color:#fff}"
+)
+# Clean up
+cat(
+  "",
+  file = fs::path("app", "styles", "main.scss")
+)

```

Comment:
Process similar for `build_js()` and `build_sass()`.

1. Check if `build_` returns default output.
2. Add minimal scss/js.
3. Check if `build_` returns the correct output.
4. Revert to no scss/js.